metadata-editor/conf/routes

```diff
@@ -1,30 +1,32 @@
-GET     /                                 controllers.EditsApi.index
-GET     /usage-rights/categories          controllers.EditsApi.getUsageRights()
+GET     /                                           controllers.EditsApi.index
+GET     /usage-rights/categories                    controllers.EditsApi.getUsageRights()
 
 # Image
-GET     /metadata/:id                     controllers.EditsController.getAllMetadata(id: String)
+GET     /metadata/:id                               controllers.EditsController.getAllMetadata(id: String)
 
-GET     /metadata/:id/archived            controllers.EditsController.getArchived(id: String)
-PUT     /metadata/:id/archived            controllers.EditsController.setArchived(id: String)
-DELETE  /metadata/:id/archived            controllers.EditsController.unsetArchived(id: String)
+GET     /metadata/:id/archived                      controllers.EditsController.getArchived(id: String)
+PUT     /metadata/:id/archived                      controllers.EditsController.setArchived(id: String)
+DELETE  /metadata/:id/archived                      controllers.EditsController.unsetArchived(id: String)
 
-GET     /metadata/:id/labels              controllers.EditsController.getLabels(id: String)
-POST    /metadata/:id/labels              controllers.EditsController.addLabels(id: String)
-DELETE  /metadata/:id/labels/*label       controllers.EditsController.removeLabel(id: String, label: String)
+GET     /metadata/:id/labels                        controllers.EditsController.getLabels(id: String)
+POST    /metadata/:id/labels                        controllers.EditsController.addLabels(id: String)
+DELETE  /metadata/:id/labels/*label                 controllers.EditsController.removeLabel(id: String, label: String)
 
-GET     /metadata/:id/metadata            controllers.EditsController.getMetadata(id: String)
-PUT     /metadata/:id/metadata            controllers.EditsController.setMetadata(id: String)
+GET     /metadata/:id/metadata                      controllers.EditsController.getMetadata(id: String)
+PUT     /metadata/:id/metadata                      controllers.EditsController.setMetadata(id: String)
 
-GET     /metadata/:id/usage-rights        controllers.EditsController.getUsageRights(id: String)
-PUT     /metadata/:id/usage-rights        controllers.EditsController.setUsageRights(id: String)
-DELETE  /metadata/:id/usage-rights        controllers.EditsController.deleteUsageRights(id: String)
+GET     /metadata/:id/usage-rights                  controllers.EditsController.getUsageRights(id: String)
+PUT     /metadata/:id/usage-rights                  controllers.EditsController.setUsageRights(id: String)
+DELETE  /metadata/:id/usage-rights                  controllers.EditsController.deleteUsageRights(id: String)
+
+POST    /metadata/:id/usage-rights-to-metadata      controllers.EditsController.applyMetadataFromUsageRights(id: String)
```

Comment:
Can we have a verb in the URI to reinforce how Naughty:tm: the endpoint is? `set-metadata-from-usage-rights` or whatnot